HelpMyStreetFE/HelpMyStreetFE/Controllers/AccountController.cs

```diff
@@ -434,6 +434,20 @@ public async Task<IActionResult> GroupVolunteers(string groupKey, CancellationTo
             return View("Index", viewModel);
         }
 
+        [Route("g/{groupKey}/reports")]
+        [HttpGet]
+        public async Task<IActionResult> Reports(string groupKey, CancellationToken cancellationToken)
+        {
+            var group = await _groupService.GetGroupByKey(groupKey, cancellationToken);
+            var user = await _authService.GetCurrentUser(cancellationToken);
+            var viewModel = await GetAccountViewModel(user, cancellationToken);

```

Comment:
TODO: Add checks that user is fully registered and has relevant role.

Reply:
done